### ISSUE TITLE
Cannot mount BPFFS on vagrant image v77

### ISSUE THREAD

**Issue by vadorovsky** [cilium/cilium#4278]
Logs:

```
May 29 16:40:10 runtime1 cilium-agent[16953]: level=debug msg="bpffs is not mounted yet; adding to list of maps to open once it is mounted" bpfMapName=cilium_tunnel_map
May 29 16:40:10 runtime1 cilium-agent[16953]: level=debug msg="bpffs is not mounted yet; adding to list of maps to remove once it is mounted" bpfMapName=tunnel_endpoint_map
May 29 16:40:10 runtime1 cilium-agent[16953]: level=debug msg="os.Hostname() returned" nodeName=runtime1
May 29 16:40:10 runtime1 cilium-agent[16953]: level=debug msg="bpffs is not mounted yet; adding to list of maps to open once it is mounted" bpfMapName=cilium_lxc
May 29 16:40:10 runtime1 cilium-agent[16953]: level=debug msg="bpffs is not mounted yet; adding to list of maps to open once it is mounted" bpfMapName=cilium_ipcache
May 29 16:40:10 runtime1 cilium-agent[16953]: level=debug msg="bpffs is not mounted yet; adding to list of maps to open once it is mounted" bpfMapName=cilium_metrics
May 29 16:40:10 runtime1 cilium-agent[16953]: level=debug msg="bpffs is not mounted yet; adding to list of maps to open once it is mounted" bpfMapName=cilium_proxy4
May 29 16:40:10 runtime1 cilium-agent[16953]: level=debug msg="bpffs is not mounted yet; adding to list of maps to open once it is mounted" bpfMapName=cilium_proxy6
May 29 16:40:10 runtime1 cilium-agent[16953]: level=debug msg="Enabling flow debug"
May 29 16:40:10 runtime1 cilium-agent[16953]: level=info msg="     _ _ _"
May 29 16:40:10 runtime1 cilium-agent[16953]: level=info msg=" ___|_| |_|_ _ _____"
May 29 16:40:10 runtime1 cilium-agent[16953]: level=info msg="|  _| | | | | |     |"
May 29 16:40:10 runtime1 cilium-agent[16953]: level=info msg="|___|_|_|_|___|_|_|_|"
May 29 16:40:10 runtime1 cilium-agent[16953]: level=info msg="Cilium 1.0.90 c96266d36caa 2018-05-18T14:16:52-07:00 go version go1.9.3 linux/amd64"
May 29 16:40:10 runtime1 cilium-agent[16953]: level=info msg="cilium-envoy  version: c96266d36caa9341c03b89e15a50031a778f03a5/1.7.0-dev/Modified/DEBUG"
May 29 16:40:10 runtime1 cilium-agent[16953]: level=info msg="clang (3.8.1) and kernel (4.9.17) versions: OK!"
May 29 16:40:10 runtime1 cilium-agent[16953]: level=info msg="linking environment: OK!"
May 29 16:40:11 runtime1 cilium-agent[16953]: level=info msg="bpf_requirements check: OK!"
May 29 16:40:11 runtime1 cilium-agent[16953]: level=debug msg="bpf_features check: Some features may be limited:"
May 29 16:40:11 runtime1 cilium-agent[16953]: level=debug msg="BPF/probes: CONFIG_CGROUP_BPF=y is not in kernel configuration"
May 29 16:40:11 runtime1 cilium-agent[16953]: level=debug msg="BPF/probes: CONFIG_LWTUNNEL_BPF=y is not in kernel configuration"
May 29 16:40:11 runtime1 cilium-agent[16953]: level=debug msg="HAVE_LPM_MAP_TYPE: Your kernel doesn't support LPM trie maps for BPF, thus switching back to using hash table for CIDR pol
May 29 16:40:11 runtime1 cilium-agent[16953]: level=debug msg="HAVE_LRU_MAP_TYPE: Your kernel doesn't support LRU maps for BPF, thus switching back to using hash table for the cilium co
May 29 16:40:11 runtime1 cilium-agent[16953]: level=fatal msg="Unable to mount BPF filesystem" error="multiple mount points detected at /sys/fs/bpf"
```

BPFFS mountpoints:

```
$ grep -i bpf /proc/mounts 
/sys/fs/bpf /sys/fs/bpf bpf rw,relatime 0 0
/sys/fs/bpf /sys/fs/bpf/xdp bpf rw,relatime 0 0
/sys/fs/bpf /sys/fs/bpf/ip bpf rw,relatime 0 0
```

**Comment by ianvernon** [comment]
@aanm : can we reduce the priority on this since the change which introduced this problem was reverted?

**Comment by vadorovsky** [comment]
Fixed by #4436
